### PR TITLE
Prepare 8.0.0 GA release

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by ruflin** [comment]
@sidz @thePanz Would be great to get your eyes on this one as I plan to tag 8.0.0 as soon as this PR is merged.

**Comment by ruflin** [comment]
Do we need any update to the branch alias? https://github.com/ruflin/Elastica/blob/8.x/composer.json#L56

**Comment by sidz** [comment]
I'd say that we don't need to have branch-alias at all @ruflin .

**Review by sidz — APPROVED** [review]
:+1:

**Comment by ruflin** [comment]
> I'd say that we don't need to have branch-alias at all @ruflin .

Should I get rid of it now or in a follow PR? @thePanz Do you remember why we had the branch alias? Because we had the ` main`  branch before?

File: CHANGELOG.md [hunk 1]
```diff
@@ -116,6 +119,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  * `Elastica\Connection\ConnectionPool`
+  * `Elastica\Connection\Strategy\CallbackStrategy`
+  * `Elastica\Connection\Strategy\RoundRobin`
+  * `Elastica\Connection\Strategy\Simple`
+  * `Elastica\Connection\Strategy\StrategyFactory`
+  * `Elastica\Connection\Strategy\StrategyInterface`
+* Removed `Elastica\Client::setLogger()` method [#2148](https://github.com/ruflin/Elastica/pull/2148)
+* Enabled `strict_types` on all classes [#2190](https://github.com/ruflin/Elastica/pull/2190)
+* Changed `Elastica\Scroll:clear` to use `scroll_id` body parameter instead of url parameter [#2211](https://github.com/ruflin/Elastica/pull/2211)
+
```
Comment (thePanz) [inline CHANGELOG.md:193]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,7 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
-## [Unreleased](https://github.com/ruflin/Elastica/compare/7.3.0...8.0.0-alpha1)
+## [8.0.0](https://github.com/ruflin/Elastica/compare/8.0.0-alpha1...8.0.0)
+
+This a major release with breaking changes. Have a look at the [Upgrade guide](https://github.com/ruflin/Elastica/blob/8.x/UPGRADE-8.0.md).
```
Comment (thePanz) [inline CHANGELOG.md]:
```suggestion
This a major release with breaking changes.
Have a look at the [Upgrade guide](https://github.com/ruflin/Elastica/blob/8.x/UPGRADE-8.0.md).
```

**Comment by thePanz** [comment]
> > I'd say that we don't need to have branch-alias at all @ruflin .
> 
> Should I get rid of it now or in a follow PR? @thePanz Do you remember why we had the branch alias? Because we had the ` main` branch before?

yes, that's the reason why we had to use the alias.

OK for me to remove it in this PR (within an additional commit)

I added just two minor suggestions, feel free to apply them @ruflin

**Review by thePanz — APPROVED** [review]

**Comment by thePanz** [comment]
LGTM @ruflin , let's squash the changelog commits before merging :+1:

**Comment by ruflin** [comment]
I was lazy and squashed it all into a single commit ;-)

**Pull request merged**